google/cloud/aiplatform/tensorboard/tensorboard.py

```diff
@@ -0,0 +1,286 @@
+
+
+class Tensorboard(base.VertexAiResourceNounWithFutureManager):
+    """Managed tensorboard resource for Vertex AI."""
+
+    client_class = utils.TensorboardClientWithOverride
+    _is_client_prediction_client = False
+    _resource_noun = "tensorboards"
+    _getter_method = "get_tensorboard"
+    _list_method = "list_tensorboards"
+    _delete_method = "delete_tensorboard"
+
+    def __init__(
+        self,
+        tensorboard_name: str,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ):
+        """Retrieves an existing managed tensorboard given a tensorboard name or ID.
+
+        Args:
+            tensorboard_name (str):
+                Required. A fully-qualified tensorboard resource name or tensorboard ID.
+                Example: "projects/123/locations/us-central1/tensorboards/456" or
+                "456" when project and location are initialized or passed.
+            project (str):
+                Optional project to retrieve tensorboard from. If not set, project
+                set in aiplatform.init will be used.
+            location (str):
+                Optional location to retrieve tensorboard from. If not set, location
+                set in aiplatform.init will be used.
+            credentials (auth_credentials.Credentials):
+                Optional custom credentials to use to retreive this Tensorboard. Overrides
+                credentials set in aiplatform.init.
+        """
+
+        super().__init__(
+            project=project,
+            location=location,
+            credentials=credentials,
+            resource_name=tensorboard_name,
+        )
+        self._gca_resource = self._get_gca_resource(resource_name=tensorboard_name)
+
+    @classmethod
+    def create(
+        cls,
+        display_name: str,
+        description: Optional[str] = None,
+        labels: Optional[Dict[str, str]] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        encryption_spec_key_name: Optional[str] = None,
+    ) -> "Tensorboard":
+        """Creates a new tensorboard.
+
+        Example Usage:
+
+            tb = aiplatform.Tensorboard.create(
+                display_name='my display name',
+                description='my description',
+                labels={
+                    'key1': 'value1',
+                    'key2': 'value2'
+                }
+            )
+
+        Args:
+            display_name (str):
+                Required. The user-defined name of the Tensorboard.
+                The name can be up to 128 characters long and can be consist
+                of any UTF-8 characters.
+            description (str):
+                Optional description of this Tensorboard.
```

Comment:
Format should be `Optional. Description of this Tensorboard`. Please apply throughout this PR.